HPXMLtoOpenStudio/resources/schema_validator.xml

```diff
@@ -0,0 +1,714 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<sch:schema xmlns:sch='http://purl.oclc.org/dsdl/schematron'>
+  <sch:ns uri='http://hpxmlonline.com/2019/10' prefix='h'/>
+  <sch:pattern>
+    <sch:rule context='//h:XMLTransactionHeaderInformation/h:Transaction'>
+      <sch:assert role='ERROR' test='contains("_create_ _update_", concat("_", text(), "_"))'>Expected value to be: 'create' or 'update'</sch:assert>
+    </sch:rule>
+    <sch:rule context='//h:ProjectStatus/h:EventType'>
+      <sch:assert role='ERROR' test='contains("_audit_ _proposed workscope_ _approved workscope_ _construction-period testing/daily test out_ _job completion testing/final inspection_ _quality assurance/monitoring_ _preconstruction_", concat("_", text(), "_"))'>Expected value to be: 'audit' or 'proposed workscope' or 'approved workscope' or 'construction-period testing/daily test out' or 'job completion testing/final inspection' or 'quality assurance/monitoring' or 'preconstruction'</sch:assert>
+    </sch:rule>
+    <sch:rule context='//h:Building/h:BuildingDetails/h:BuildingSummary/h:Site/h:SiteType'>
+      <sch:assert role='ERROR' test='contains("_rural_ _suburban_ _urban_", concat("_", text(), "_"))'>Expected value to be: 'rural' or 'suburban' or 'urban'</sch:assert>
+    </sch:rule>
+    <sch:rule context='//h:Building/h:BuildingDetails/h:BuildingSummary/h:Site/h:FuelTypesAvailable/h:Fuel'>
+      <sch:assert role='ERROR' test='contains("_electricity_ _renewable electricity_ _natural gas_ _renewable natural gas_ _fuel oil_ _fuel oil 1_ _fuel oil 2_ _fuel oil 4_ _fuel oil 5/6_ _district steam_ _district hot water_ _district chilled water_ _solar hot water_ _propane_ _kerosene_ _diesel_ _coal_ _anthracite coal_ _bituminous coal_ _coke_ _wood_ _wood pellets_ _combination_ _other_", concat("_", text(), "_"))'>Expected value to be: 'electricity' or 'renewable electricity' or 'natural gas' or 'renewable natural gas' or 'fuel oil' or 'fuel oil 1' or 'fuel oil 2' or 'fuel oil 4' or 'fuel oil 5/6' or 'district steam' or 'district hot water' or 'district chilled water' or 'solar hot water' or 'propane' or 'kerosene' or 'diesel' or 'coal' or 'anthracite coal' or 'bituminous coal' or 'coke' or 'wood' or 'wood pellets' or 'combination' or 'other'</sch:assert>
+    </sch:rule>
+    <sch:rule context='//h:Building/h:BuildingDetails/h:BuildingSummary/h:BuildingOccupancy/h:NumberofResidents'>
+      <sch:assert role='ERROR' test='number(.) &gt;= 0'>Expected value to be greater than or equal to 0 for xpath: </sch:assert>
```

Comment:
Why the "for xpath" ending? Can we not simply have the message be "Expected value to be greater than or equal to 0"? There are other cases where the message is simply ">Expected value to be: 'create' or 'update'".